src/battle-dex.ts

```diff
@@ -215,6 +215,8 @@ const Dex = new class implements ModdedDex {
 	readonly statNames: ReadonlyArray<StatName> = ['hp', 'atk', 'def', 'spa', 'spd', 'spe'];
 	readonly statNamesExceptHP: ReadonlyArray<StatNameExceptHP> = ['atk', 'def', 'spa', 'spd', 'spe'];
 
+	pokeballs: string[] = null!;
```

Comment:
```suggestion
	pokeballs: string[] | null = null;
```

?

Reply:
I use `null!` for "null during initialization only". This one stays null until `getPokeballs` is called.